content/strategy-goals/strategy/code-graph/search/product.md

```diff
@@ -10,37 +10,71 @@ Quicklinks:
 - [Search Product backlog](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fsearch-product/)
 - [Roadmap](https://github.com/orgs/sourcegraph/projects/214/views/34?filterQuery=owning-org%3A%22Code+Graph%22+type%3ARoadmap+owning-team%3A%22Search+product%22)
 
+## Goals
```

Comment:
This goals section feels like it might be better in the "what's next and why" section for a couple reasons:

- The page flow won't bounce between future to past to future
- Some of the needed context will be closer (i.e., how are you going to achieve these goals is in the whats next and why section.)

My suggestion would be to start the what's next and why section with what your goals are, and then go from there.

Reply:
A second thought, the goals listed here are more like principles or guidelines than goals. Continuing to differentiate, providing compelling solutions for important needs, and maintaing room for experimentation aren't particularly measurable or have a point where you could say they are done. They are pretty open ended aspirations.

Reply:
Maybe with this edit we could keep the page flow as-is?

```suggestion
## Guiding principles
```